lib/logstash/event.rb

```diff
@@ -16,6 +16,13 @@
 class LogStash::ShutdownEvent; end
 class LogStash::FlushEvent; end
 
+module LogStash
```

Comment:
Good suggestion but unfortunately we cannot do this without updating all plugins which rely on `LogStash::SHUTDOWN`, see #2863 which is exactly what happened. I suggest you create a new issue and we can tackle this in >= 1.6. It could be done at the same time we do #2872